code401-challenges/src/main/java/code401/challenges/stacksandqueues/Queue.java

```diff
@@ -0,0 +1,60 @@
+package code401.challenges.stacksandqueues;
+
+import code401.challenges.linkedlist.Node;
+
+public class Queue {
+    Node front;
+    Node back;
+
+    public Queue(){
+        this.front = null;
+        this.back = null;
+    }
+
+    //method to add a new node to the back of the queue
+    public void enqueue(int value){
+        Node current = new Node();
+        current.setValue(value);
+        if(front == null){
+            back = current;
+            front = back;
+            front.setNext(null);
```

Comment:
I think this line is redundant. When you initialized current its next was already `null`